assets/section-image-banner.css

```diff
@@ -279,13 +285,26 @@
   max-width: 45rem;
 }
 
-.banner--text-align-right .banner__buttons--multiple { 
-  justify-content: flex-end;
+@media screen and (max-width: 749px) {
+  .banner--text-align-mobile-right .banner__buttons--multiple {
+    justify-content: flex-end;
+  }
+
+  .banner--text-align-mobile-center .banner__buttons--multiple > * {
+    flex-grow: 1;
+    min-width: 22rem;
+  }
 }
 
-.banner--text-align-center .banner__buttons--multiple > * { 
-  flex-grow: 1;
-  min-width: 22rem;
+@media screen and (min-width: 750px) {
+  .banner--text-align-center .banner__buttons--multiple > * {
+    flex-grow: 1;
+    min-width: 22rem;
+  }
```

Comment:
Should this be there for desktop ? I find the buttons a little too big imo: 
![](https://screenshot.click/21-11-pitw7-h3dbh.png)

Reply:
This is because of the `min-width: 22rem;` , which was set previous to this PR

Reply:
The rest of the PR looked good, just curious to hear what designers think about this layout/look when centered on desktop. 
I talked to Wik already and I think she is having a look this afternoon.

Reply:
cc @wiktoriaswiecicka @melissaperreault

Reply:
Confirmed with Wik that this is expected :)